pkg/keycloak/keycloak.go

```diff
@@ -436,6 +435,38 @@ func (h *Handler) reconcileClients(kc *v1alpha1.Keycloak, kcClient KeycloakInter
 	return nil
 }
 
+func (h *Handler) reconcileUsers(kc *v1alpha1.Keycloak, kcClient KeycloakInterface, objRealm *v1alpha1.KeycloakRealm) error {
+	logrus.Info("reconciling users")
+
+	users, err := kcClient.ListUsers(objRealm.Realm)
+	if err != nil {
+		return err
+	}
+
+	kcUsers := map[string]*v1alpha1.KeycloakUser{}
+	for i := range users {
+		kcUsers[users[i].UserName] = users[i]
+	}
+
+	userPairsList := map[string]*v1alpha1.KeycloakUserPair{}
+	for i := range objRealm.Users {
+		user := objRealm.Users[i]
```

Comment:
Can we either remove the "realm" variable, or have it point to the original address:

```
		user := &objRealm.Users[i]
		userPairsList[user.UserName] = &v1alpha1.KeycloakUserPair{
			ObjUser: user,
			KcUser:  kcUsers[user.UserName],
		}
```